### PR TITLE
Add emails about the upcoming December Tech Day

### PR DESCRIPTION
**Pull request by PeterJCLaw**
- cancel the London one
- advertise the Virtual one we're doing instead

Depends on https://github.com/srobo/website/pull/463

### CODE REVIEW

File: SR2023/2022-11-23-cancel-london-december-tech-day.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Instead we are planning to run a [Virtual Tech Day][virtual-tech-day] in
+[Discord](https://studentrobotics.org/docs/team_admin/discord) on the same day,
+so that teams who are able to work in another location are still able to get
+support to progress their robots.
+
+A voice channel will be created shortly before the event starts, where you and
+your competitors can join and chat with us. We have a few group activities
+planned which will take place here.
+
+We look forward to seeing you all then.
```
Comment (Adimote) [inline SR2023/2022-11-23-cancel-london-december-tech-day.md:23]: [nit] maybe make this a tiny bit informal by using an excalamation mark
```suggestion
We look forward to seeing you all then!
```

File: SR2023/2022-11-23-virtual-december-tech-day.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+support, and assist with your strategy ideas.
+
+We see a strong correlation between teams to attend Tech Days and performance in
+the competition.
+
+A voice channel will be created shortly before the event starts, where you and
+your competitors can join and chat with us. We have a few group activities
+planned which will take place here.
+
+We look forward to seeing you all then.
```
Comment (Adimote) [inline SR2023/2022-11-23-virtual-december-tech-day.md:26]: [nit] maybe make this a tiny bit informal by using an excalamation mark?
```suggestion
We look forward to seeing you all then.
```

File: SR2023/2022-11-23-virtual-december-tech-day.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+Hello,
+
+We will be hosting a [virtual Tech Day on the 3rd December][virtual-tech-day] in
+our [Discord](https://studentrobotics.org/docs/team_admin/discord) server.
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They’re also an opportunity to see how other
+teams are doing or get more direct help with your robots.
```
Comment (Adimote) [inline SR2023/2022-11-23-virtual-december-tech-day.md:13]: [nit] we're mixing "their" and "your" here

```suggestion
teams are doing or get more direct help with their robots.
```

**Review by Adimote — DISMISSED** [review]

File: SR2023/2022-11-23-virtual-december-tech-day.md (hunk 3 above)
Comment (PeterJCLaw) [reply SR2023/2022-11-23-virtual-december-tech-day.md:13]: Think this is actually fine, as discussed.

**Review by Adimote — DISMISSED** [review]

**Pull request merged**